x/emissions/keeper/topic_activation.go

```diff
@@ -338,19 +337,19 @@ func (k *Keeper) AttemptTopicReactivation(ctx context.Context, topicId TopicId)
 	// Remove current active topic from block
 	err = k.removeCurrentTopicFromBlock(ctx, topicId, currentBlock)
 	if err != nil {
-		sdkCtx.Logger().Warn(fmt.Sprintf("Failed to remove current active topic from block %d", topicId))
+		sdkCtx.Logger().Warn("Failed to remove current active topic", "topicId", topicId, "block", currentBlock)
```

Comment:
```suggestion
		sdkCtx.Logger().Warn("Failed to remove current active topic", "topicId", topicId, "block", currentBlock, "error", err)
```

Maybe worth adding the err in the log here